plugin/src/main/resources/io/github/noeppi_noeppi/tools/modgradle/plugins/packdev/install_server.py

```diff
@@ -26,7 +34,8 @@ def setup_server():
     print('Installing Forge')
     mcv = mods[0][0]
     mlv = mods[0][1]
-    request = Request(f'https://maven.minecraftforge.net/net/minecraftforge/forge/{mcv}-{mlv}/forge-{mcv}-{mlv}-installer.jar')
+    request = Request(
+        f'https://maven.minecraftforge.net/net/minecraftforge/forge/{mcv}-{mlv}/forge-{mcv}-{mlv}-installer.jar')
```

Comment:
Multiline uglyness. In that case I don't really care about PEP8, so if you don't admire that formatting, consider making it one line.